### PR TITLE
Namibia Safaris Mobile and Desktop version

### PR DESCRIPTION
**Pull request by rtonata88**
This PR is the complete version of the Namibia Safaris website. The website has the following sections

- Home
- Main activities for Namibia Safaris
- Accommodation partners
- Main destinations
- About page

The website is full responsive, designed for both small screens and large screens.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
```
Comment (D4nielJ) [inline index.html:1]: - [x] It seems your mobile menu is experiencing a bug. I can't make it open. Let's try to fix that.

Comment (D4nielJ) [reply index.html:1]: - [x] Please don't use `async` and `defer` on the same JS script. These two are opposites, so I'm not exactly sure of the behavior they will present. In this case, use only `defer`, as you want the script to load once your HTML has finished loading. [You can read more about defer and async attributes on script tag here.](https://www.w3schools.com/tags/att_script_defer.asp)

Comment (D4nielJ) [reply index.html:1]: - [x] Please make sure that your script executes when your HTML has finished loading. To do this simply wrap all of your JavaScript code in a `main` function, then at the end of your page place this line `window.onload = main;`

Comment (D4nielJ) [reply index.html:1]: - [x] I think you could try to style the images from the `Accommodation Partners` section. Some of them are currently flowing out of the page, causing a layout with a lot of blank space on the side.

Reference:
image

File: about.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
```
Comment (D4nielJ) [inline about.html:1]: - [x] Just as in your index.html page, let's make sure that images are not flowing out of the page on the about section.

Reference:
image

Comment (D4nielJ) [reply about.html:1]: - [x] Let's try to be more consistent with the design provided by not displaying the lists of partners on the `about` section and changing the footer to work with a dark background.

Yours:
image


Template:
image

Comment (D4nielJ) [reply about.html:1]: - [x] Let's link the logo in your navbar to the home page. This ensures that the user can easily navigate through every place of your website, resulting in better User Experience.

- [x] Let's also include a link to the home page on the mobile menu, so users in mobile can navigate just as easily.

Comment (D4nielJ) [reply about.html:1]: - [x] Please, make sure that your mobile menu is working on both the index and the about section.

Comment (D4nielJ) [reply about.html:1]: - [x] Lastly, but not least. Please include in your README file credits to the original design idea, as required by the [Creative Commons License.](https://creativecommons.org/licenses/)

**Review by D4nielJ — CHANGES_REQUESTED** [review]
## Hello @rtonata88, 

You're doing great! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Please, when asking for reviews, make sure that all of the features required from Microverse are present on the branch where you're asking for a Pull Request. This is what allows reviewers to use the full extent of the GitHub reviewing feature and comment on every line of code that has been written.

## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VanessaAoki — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED :yellow_circle: 
Hi @rtonata88, 

Great work on making the changes requested by the previous reviewer, but there is still one last which isn't addressed yet.


 ## Required Changes ♻️
- [x] As requested by the previous CR, you should consider adding the proper credits to the design author. You could add a simple `Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07).` in your acknowledge's section in the REDME file, for instance.


really close

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rtonata88** [comment]
I've added a license

**Comment by VanessaAoki** [comment]
Great job adding the license. But if you read it, it determines that the author should be credited, as you can see here:

image

**Comment by rtonata88** [comment]
Thanks will do.

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## Thank you for implementing all the project requirements, you've done a great job :trophy: There is nothing else to say other than... it's time to merge it :champagne:  :tada: Congratulations

Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**